client/components/layout/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+export default class Header extends Component {
```

Comment:
Since you're not dealing with state in this component, it would probably be better of as a function component accepting the props you need:

```
function Header({title}) {
return (
      <main className="header-container container-fluid">
        <section className="row">
          <div className="col d-flex justify-content-center">
            <h5>{title}</h5>
          </div>
        </section>
      </main>
}
```